openslides_backend/action/actions/motion/create.py

```diff
@@ -154,7 +156,12 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
             forbidden_fields.add("attachment_ids")
 
         perm = Permissions.Motion.CAN_MANAGE
-        if not has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
+        if (
+            self.check_perm_and_delegator_restriction(
+                perm, "users_forbid_delegator_as_submitter", [instance["meeting_id"]]
+            )
+            == []
+        ):
```

Comment:
same here

Reply:
It's necessary to retain old functionality